### PR TITLE
Change to example webapp

### PR DESCRIPTION
**Pull request by thusithathilina**

### CODE REVIEW

File: modules/samples/example/src/main/resources/carbon/authentication/login.jsp [hunk 1]
```diff
@@ -18,23 +18,36 @@
...
-        try {
-            CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
-            UserRealm realm = context.getUserRealm();
-            status = realm.getUserStoreManager().authenticate(username, password);
-        } catch (UserStoreException e) {
-            e.printStackTrace();
+    try {
+        CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+        UserRealm realm = context.getUserRealm();
+        if(role == null){
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/authentication/login.jsp]: Formatting issue in the if block

File: modules/samples/example/src/main/resources/carbon/authentication/login.jsp.html [hunk 2]
```diff
@@ -15,30 +15,53 @@
...
+<a name="line6"> 6</a> &lt;%
+<a name="line7"> 7</a>     String username = request.getParameter(&quot;username&quot;);
+<a name="line8"> 8</a>     String password = request.getParameter(&quot;password&quot;);
+<a name="line9"> 9</a>     String role = request.getParameter(&quot;role&quot;);
+<a name="line10">10</a>     boolean status = false;
+<a name="line11">11</a>
+<a name="line12">12</a>     try {
+<a name="line13">13</a>         CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+<a name="line14">14</a>         UserRealm realm = context.getUserRealm();
+<a name="line15">15</a>         if(role == null){
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/authentication/login.jsp.html]: same formatting issue as above

File: modules/samples/example/src/main/resources/carbon/authentication/login.jsp [hunk 3]
```diff
@@ -18,23 +18,36 @@
...
+            }
         }
+    } catch (UserStoreException e) {
+        e.printStackTrace();
     }
+
     if (status) {
         session.setAttribute("logged-in", "true");
         session.setAttribute("username", username);
+        if(role != null){
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/authentication/login.jsp]: formatting issue

File: modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+<head></head>
+<body>
+<h2>WSO2 Carbon Role Manage Usage Demo</h2>
+
+<%
+    CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+    UserRealm realm = context.getUserRealm();
+    String username = request.getParameter("username");
+
+    if(request.getParameter("add") != null || request.getParameter("remove") != null){
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp]: formatting issue

File: modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp.html [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+<a name="line7">  7</a> &lt;html&gt;
+<a name="line8">  8</a> &lt;head&gt;&lt;/head&gt;
+<a name="line9">  9</a> &lt;body&gt;
+<a name="line10"> 10</a> &lt;h2&gt;WSO2 Carbon Role Manage Usage Demo&lt;/h2&gt;
+<a name="line11"> 11</a>
+<a name="line12"> 12</a> &lt;%
+<a name="line13"> 13</a>     CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+<a name="line14"> 14</a>     UserRealm realm = context.getUserRealm();
+<a name="line15"> 15</a>     String username = request.getParameter(&quot;username&quot;);
+<a name="line16"> 16</a>
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp.html]: formatting issue in line 41, 46, 47, 49, 53, 63, 66.

File: modules/samples/example/src/main/resources/carbon/usermgt/index.jsp [hunk 6]
```diff
@@ -23,21 +23,35 @@
...
+        String username = request.getParameter("username");
+        if(username != null && !username.trim().isEmpty()){
+            CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+            UserRealm realm = context.getUserRealm();
+            if (realm.getUserStoreManager().isExistingUser(username)) {
+                realm.getUserStoreManager().deleteUser(username);
+            } else {
+%> <p><b>The user <%=username%> does not exists</b></p> <%
+            }
         }
```
Comment (Nipuni) [inline modules/samples/example/src/main/resources/carbon/usermgt/index.jsp]: formatting issues in line 26, 38, 40,

File: modules/samples/example/src/main/resources/carbon/index.html [hunk 7]
```diff
@@ -1,143 +1,162 @@
...
-        </td>
-    </tr>
-</table>
-
-</body>
-</html>
+<!doctype html public "-//w3c//dtd html 4.0 transitional//en">
+<html>
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (callkalpa) [inline modules/samples/example/src/main/resources/carbon/index.html]: Update the license with year

File: modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+        String role = request.getParameter("role");
+        if (username != null && username.trim().length() > 0) {
+            if (realm.getUserStoreManager().isExistingUser(username) && realm.getUserStoreManager().isExistingRole(role)) {
+                List<String> rolesList = new ArrayList<String>(Arrays.asList(realm.getUserStoreManager().getRoleListOfUser(username)));
+                if(request.getParameter("add") != null){
+                    if(!rolesList.contains(role)){
```
Comment (callkalpa) [inline modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp]: Formatting issue

File: modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+                        realm.getUserStoreManager().updateRoleListOfUser(username, new String[]{role}, new String[]{});
+                    } else {
+%> <p><b>The user <%=username%> does not have <%=role%> </b></p> <%
+                    }
+                }
+            } else {
+%> <p><b>The user <%=username%> or <%=role%> does not exist</b></p> <%
+            }
+        }
+    } else if(request.getParameter("view") != null){
```
Comment (callkalpa) [inline modules/samples/example/src/main/resources/carbon/rolemgt/index.jsp]: formatting issue

File: modules/samples/example/src/main/resources/carbon/rolemgt/source.html [hunk 10]
```diff
@@ -0,0 +1,33 @@
+<html>
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (callkalpa) [inline modules/samples/example/src/main/resources/carbon/rolemgt/source.html]: update year

File: modules/samples/example/src/main/resources/carbon/usermgt/index.jsp [hunk 11]
```diff
@@ -23,21 +23,35 @@
...
 <%
-    String username = request.getParameter("username");
-    String pass = request.getParameter("password");
-    if (username != null && username.trim().length() > 0) {
-        CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
-        UserRealm realm = context.getUserRealm();
-        if (!realm.getUserStoreManager().isExistingUser(username)) {
-            realm.getUserStoreManager().addUser(username, pass, null, null, null);
-        } else {
+    if(request.getParameter("add") != null){
```
Comment (arunasujith) [inline modules/samples/example/src/main/resources/carbon/usermgt/index.jsp]: formatting issue

File: modules/samples/example/src/main/resources/carbon/usermgt/index.jsp [hunk 12]
```diff
@@ -23,21 +23,35 @@
...
+            UserRealm realm = context.getUserRealm();
+            if (!realm.getUserStoreManager().isExistingUser(username)) {
+                realm.getUserStoreManager().addUser(username, pass, null, null, null);
+            } else {
 %> <p><b>The user <%=username%> already exists</b></p> <%
+            }
+        }
+    } else if(request.getParameter("delete") != null){
+        String username = request.getParameter("username");
+        if(username != null && !username.trim().isEmpty()){
```
Comment (arunasujith) [inline modules/samples/example/src/main/resources/carbon/usermgt/index.jsp]: formatting issue

File: modules/samples/example/src/main/resources/carbon/usermgt/index.jsp [hunk 13]
```diff
@@ -23,21 +23,35 @@
...
+        if (username != null && username.trim().length() > 0) {
+            CarbonContext context = CarbonContext.getThreadLocalCarbonContext();
+            UserRealm realm = context.getUserRealm();
+            if (!realm.getUserStoreManager().isExistingUser(username)) {
+                realm.getUserStoreManager().addUser(username, pass, null, null, null);
+            } else {
 %> <p><b>The user <%=username%> already exists</b></p> <%
+            }
+        }
+    } else if(request.getParameter("delete") != null){
```
Comment (arunasujith) [inline modules/samples/example/src/main/resources/carbon/usermgt/index.jsp]: formatting issue

**Pull request merged**